product_docs/docs/pem/8/pem_online_help/06a_toc_pem_barman/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: "Monitoring Barman"
+---
+
+Postgres Enterprise Manager (PEM) is designed to assist database administrators, system architects, and performance analysts when administering, monitoring, and tuning PostgreSQL and Advanced Server database servers.
+
+Barman (Backup and Recovery Manager) is an open-source administration tool for remote backups and disaster recovery of PostgreSQL servers in business-critical environments. It relies on PostgreSQL’s Point In Time Recovery technology, allowing DBAs to remotely manage a complete catalogue of backups and the recovery phase of multiple remote servers – all from one location. For more information about Barman, please visit the EDB website at:
+
+  https://www.enterprisedb.com/docs/supported-open-source/barman/
+
+From PEM version 8.4 onwards, you can monitor a Barman server through PEM console. PEM provides a user-friendly interface that allows you to monitor your Barman server via PEM console.
+
+Before you manage a Barman server through PEM console, you must ensure that your system meets certain requirements.  For more information, see:
+
+-   [Prerequisites for monitoring Barman](01_managing_barman_prerequisites)
```

Comment:
add this to the line above.

Reply:
Make the same changes to the sentences starting on line 17 and 21.

Reply:
done changes in a separate commit